api/specs/web-server/_folders.py

```diff
@@ -51,6 +48,8 @@ async def create_folder(_body: CreateFolderBodyParams):
 )
 async def list_folders(
     params: Annotated[PageQueryParameters, Depends()],
+    folder_id: FolderID | None = None,
```

Comment:
so If I get it right:
- both None, means you get the root workspace projects?
- is this documented somewhere?